### PR TITLE
Return vector with TaylorModel1 from validated_integ

### LINKED ISSUE #37: Return TaylorModel1 from validated_integ
As we discussed elsewhere, since we have that info for each time step covered, it is a good idea to return it.

### PR DESCRIPTION
**Pull request by lbenet**
Fixes #37

### CODE REVIEW

File: src/validatedODEs.jl [hunk 1]
```diff
@@ -215,19 +215,20 @@ function validated_step!(f!, t::Taylor1{T},
...
             for i in eachindex(x)
-                auxTM = fp_rpa( TaylorModelN(x[i](0..δt), rem[i]+Δ[i], q0, q0box) )
-                xTMN[i] = absorb_remainder(auxTM)
-                rem[i] = remainder(xTMN[i])
+                xTMN[i] = fp_rpa( TaylorModelN(x[i](0..δt), rem[i], q0, q0box) )
+
                 # If remainder is still too big, do it again
-                if mag(rem[i]) > 1.0e-10
+                j = 0
+                while (j < 10) && (mag(rem[i]) > 1.0e-10)
```
Comment (mforets) [inline src/validatedODEs.jl]: was the intention to increase `j` in the while loop or something?

Comment (lbenet) [reply src/validatedODEs.jl]: Thanks! Good catch! I'll push a corrected version.

**Comment by lbenet** [comment]
I think this is ready to be merged.

**Comment by lbenet** [comment]
Rebased to current master (after #44).

**Pull request merged**